src/core/src/Context.cpp

```diff
@@ -754,4 +831,23 @@ RenderStatistics Context::getRenderStatistics() const {
     return renderStatistics;
 }
 
+void Context::addGlobeAnchorToPrim(const pxr::SdfPath& path) {
```

Comment:
From offline discussion: are globe anchors added to the registry when loaded from a file? Do we need to watch for globe anchors in `processPrimAdded` instead of `addGlobeAnchorToPrim`?

Reply:
As discussed during stand-up, I will be fixing this in a subsequent PR. I opened #489 for this.